### DISCUSSION TITLE
Should cache be a part of the backup? Or should restore be able to initiate a re-scan?

### DISCUSSION THREAD

**Discussion by davralin** [photoprism/photoprism#1880] [Q&A] [answered]
Greetings,
I'm still evaluating photoprism - so far it looks impressive! :-)

I'm a little unsure regarding the cache-directory, coupled with the backup/restore-command.

I have all my originals stored elsewhere, so my Originals-folder is mounted read-only (with `PHOTOPRISM_READONLY` set to `true`).

That means backup for me, should really only be the database, to restore face-classification and other things.

But I also notice that I have a good amount of gb's in the cache-folder - and I'm a little bit unsure how to handle that in a backup/restore-scenario.

Which leads me to the question in the topic.

Should cache be something that's included in a backup? Not necessarily using the `backup`-command - but perhaps just mentioned in the instructions.

Or should there be something in the restore-part, that either describes how to re-init the cache, or does it automatically?

I have a feeling that the obvious answer here is that a re-scan after a restore will re-create the cache here, but I couldn't find that documented somewhere - and assumptions and restores isn't something I like doing. :-)

It might also be that people would rather backup a "useless" cache, than re-scan the entire library.

**Comment by graciousgrey** [comment, marked as answer]
The cache folder (https://docs.photoprism.app/user-guide/advanced/folders/#cache) contains thumbnails and json metadata files. 
Those can be recreated by photoprism, so if you don't mind doing a rescan they must not be included in a backup.

You find some more information on backup strategies in this discussion: https://github.com/photoprism/photoprism/discussions/772

**Reply by davralin** [reply]
#772 was dead/old when I started this discussion - so I didn't want to rescurrect it.

The only mention of cache in there was left unanswered to.

I can move this discussion over there, now that you have mentioned it and someone else rescurrected it.
